.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       with:
         node-version: 14.x
     - run: npm install
-    - run: npm run lint
+    - run: npm run mdlint
```

Comment:
So eslint isn't running in the pipeline? Is this intended?

Reply:
eslint is always the first step of "npm run test". See https://github.com/ni/vscode-ni-python-dataplugins/blob/master/src/test/runTest.ts#L15

I didn't know that either but found out that the linting was called twice - we can save that.

Reply:
sry wanted to show you this link: https://github.com/ni/vscode-ni-python-dataplugins/runs/2700473351?check_suite_focus=true#step:5:14